### PR TITLE
Geography cleanup

### PR DESCRIPTION
**Pull request by zradoslav**
Cleaning up popup menu builder for Geography plugins and class members as well - they're not used at class level, thus can be substituted with corresponding local variables

### CODE REVIEW

**Comment by Nick-Hall** [comment]
Rebased.

File: gramps/plugins/lib/maps/geography.py [hunk 1]
```diff
@@ -397,8 +394,7 @@ def build_nav_menu(self, osm, event, lat, lon):
         add_item = Gtk.MenuItem(label=title)
         menu.append(add_item)
 
-        self.changemap = Gtk.Menu()
```
Comment (Nick-Hall) [inline gramps/plugins/lib/maps/geography.py:409]: We had an issue in the past, where a menus failed to display if they were stored in local variables.  I'm not sure if this has been fixed on all supported Gtk versions yet.

**Comment by Nick-Hall** [comment]
@SNoiraud What is your opinion of this PR?

**Comment by SNoiraud** [comment]
@Nick-Hall If I remember, I tested this PR and it worked for me. The test was on my ubuntu 18.04.

File: gramps/plugins/lib/maps/geography.py (hunk 1 above)
Comment (SNoiraud) [reply gramps/plugins/lib/maps/geography.py:409]: When I tested this PR, I didn't see problems. I suppose this has been solved.

**Pull request merged**